src/components/transactions.js

```diff
@@ -55,16 +55,11 @@ export default class Transactions extends React.Component {
         return inflow
     }
 
-    openForm() {
-        document.getElementById("add-transaction").style.display = "block";
-    }
-
     render() {
         return (
             <div class="transactions">
-                <button onclick="openForm()">ADD TRANSACTION</button>
-                <p>Inflow: ${this.calcInflow(this.state.transactions)}</p>
-                <p>Outflow: ${this.calcOutflow(this.state.transactions)}</p>
+                <p><b>Inflow:</b> ${this.calcInflow(this.state.transactions)}</p>
```

Comment:
https://www.w3schools.com/tags/tag_b.asp

 According to the HTML5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate.